frodo.js

```diff
@@ -1,44 +1,44 @@
-const capitalizedString = string.charAt(0).toUpperCase() + string.slice(1);
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.